README.rst

```diff
@@ -97,6 +102,22 @@ You may also find the `authentication document`_ shared by all the
 .. _Authentication section: https://google-cloud-python.readthedocs.io/en/latest/google-cloud-auth.html
 .. _authentication document: https://github.com/GoogleCloudPlatform/gcloud-common/tree/master/authentication
 
+.. _Alpha: #versioning
+.. _Beta: #versioning
+
+Versioning
```

Comment:
This section doesn't belong here. We should just link to the GCP Alpha/Beta distinction (if it exists). @jonparrott?